src/docs/whats-new-archive.md

```diff
@@ -11,13 +11,167 @@ join the [flutter-announce][] Google group.
 
 [flutter-announce]: https://groups.google.com/forum/#!forum/flutter-announce
 
+
+## **May 6, 2020, Work-From-Home Edition**
```

Comment:
Small nit, but the headings are extra bold with the `##` and `**` applied. Is that by design?

Reply:
No, it's cause I'm lazy.  :). Fixed